src/components/Homapage.js

```diff
@@ -8,53 +8,42 @@ import {
   PinImage,
   SaveButton,
 } from "../styles/Homepage.module";
+import {initializeApp} from "firebase/app";
+import {getFirestore, collection, getDocs} from "firebase/firestore";
 
-import chicken from "../test-images/chicken.jpg";
-import kitty from "../test-images/kitty.jpg";
-import wolf from "../test-images/wolf.jpg";
-import flower from "../test-images/flower.jpg";
-import bear from "../test-images/bear.jpg";
-import cherry from "../test-images/cherry.jpg";
-import death from "../test-images/death.jpg";
-import frog from "../test-images/frog.jpg";
-import guitarfrog from "../test-images/guitar-frog.jpg";
-import hands from "../test-images/hands.jpg";
-import raccoon from "../test-images/raccoon.jpg";
-import trex from "../test-images/t-rex.jpg";
-import uglyflower from "../test-images/ugly-flower.jpg";
-import skateboard from "../test-images/skateboard.jpg";
-import duck from "../test-images/duck.jpg";
-
-const mockAllPins = [
-  {pinImage: chicken},
-  {pinImage: kitty},
-  {pinImage: wolf},
-  {pinImage: flower},
-  {pinImage: bear},
-  {pinImage: cherry},
-  {pinImage: death},
-  {pinImage: frog},
-  {pinImage: guitarfrog},
-  {pinImage: hands},
-  {pinImage: raccoon},
-  {pinImage: trex},
-  {pinImage: uglyflower},
-  {pinImage: skateboard},
-  {pinImage: duck},
-];
+const firebaseConfig = {
+  apiKey: process.env.REACT_APP_FIREBASE_APIKEY,
+  authDomain: process.env.REACT_APP_FIREBASE_DOMAIN,
+  projectId: process.env.REACT_APP_FIREBASE_PROJECTID,
+  storageBucket: process.env.REACT_APP_FIREBASE_STORAGEBUCKET,
+  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGINGSENDERID,
+  appId: process.env.REACT_APP_FIREBASE_APPID,
+  measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENTID,
+};
 
 function Homapage() {
   const [isShowAddPin, setIsShowAddPin] = useState(false);
   const [pins, setPins] = useState();
```

Comment:
initialize with empty array.

Reply:
Modified; however, I've used this state to determine if certain function should run or not, so I've changed the condition to '{ pins.length>0 && ... }'.

src/ components/ Homepage.js line : 17, 45